.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
-## SP 800-63-3 Public Comment Period has closed!
+## SP 800-63-3 Has Been Published
 
-The editorial team is in the process of incorporating responses to the comments and completing editing of the documents. All further issues received will be automatically closed.
+The SP 800-63-3 document suite has been finalized and published. It is publicly available at https://pages.nist.gov/800-63-3/
 
-Thank you for your interest in supporting and improving NIST SP 800-63-3.
+As the document has been finalized in its current revision, requests for normative changes will be rejected or targeted to a future revision of the document suite. Editorial changes and internal clarifications may be incorporated in an errata set for this revision. Additional context and discussion may be addressed by the FAQ (https://pages.nist.gov/800-63-FAQ/) or other publications and resources. 
```

Comment:
I'd suggest saying that the changes will be targeted and adjudicated for the next revision of the document suite. I don't think we should give the impression that we're going to reject anything out of hand.